app/cdap/components/GridTable/utils.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2017-2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React from 'react';
+import { IExecuteAPIResponse } from './types';
+
+export const convertNonNullPercent = (gridData: IExecuteAPIResponse, key: string, nonNullValue) => {
+  const lengthOfData: number = gridData?.values.length;
```

Comment:
gridData? means that gridData can possibly be undefined which isn't reflected in the type above

Reply:
Added type